# Description: Manual typos
#  add missing backslashes before LaTeX ldots command.
# Author: Arjan Oosting <arjanoosting@home.nl>
Index: frown/Manual/Manual.lhs
===================================================================
--- frown.orig/Manual/Manual.lhs	2016-02-08 11:17:17.969450404 +0100
+++ frown/Manual/Manual.lhs	2016-02-08 11:17:17.969450404 +0100
@@ -523,7 +523,7 @@
 source file (@Paren1.lg@\footnote{The source files of the examples are
 located in the directory @Manual/Examples@.}) defines the language of
 well-balanced parentheses. The specification of the grammar is
-enclosed in special curly braces `|%{ ldots }%|'. The remainder
+enclosed in special curly braces `|%{ \ldots }%|'. The remainder
 contains Haskell source code, that is, a module header and a function
 declaration.
 %
@@ -542,7 +542,7 @@
 By convention, the first nonterminal is also the start symbol of
 the grammar (this default can be overwritten, see~Sec.~\ref{sec:multiple}).
 
-Productions have the general form |n : v_1, ldots, v_k;| where |n| is
+Productions have the general form |n : v_1, \ldots, v_k;| where |n| is
 a nonterminal and |v_1|, \ldots, |v_k| are symbols. Note that the
 symbols are separated by commas and terminated by a semicolon. The
 mandatory trailing semicolon helps to identify so-called
@@ -816,7 +816,7 @@
 pure one. It is, however, also possible to provide a monadic action
 that \emph{computes} the value of the attribute. The computation lives
 in the underlying parsing monad. Monadic actions are enclosed in `|{%
-ldots }|' braces and have type |m t| where |m| is the type of the
+\ldots }|' braces and have type |m t| where |m| is the type of the
 underlying monad and |t| is the type of attributes.
 
 As an example, the following variant of the desktop calculator
@@ -1380,7 +1380,7 @@
 <                               |  Ident "then"  as "then"
 <                               |  Ident "else"  as "else"
 <                               |  Ident {String};
-<                               |  ldots
+<                               |  \ldots
 
 Note that keywords are declared just by listing them before the
 general pattern for identifiers.
@@ -1391,7 +1391,7 @@
 following example.
 
 < Terminal                      =  guard { isAlpha } as "alpha"
-<                               |  ldots
+<                               |  \ldots
 
 A guard is introduced by the keyword |guard|, followed by its Haskell
 definition, followed by the mandatory shortcut. The shortcut can then
